src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/segmentgroup/LabResultTest.java

```diff
@@ -98,6 +99,26 @@ void testGetLaboratoryInvestigationResult() {
             .contains(Optional.of(MeasurementValueComparator.LESS_THAN));
     }
 
+    @Test
+    void testGetLaboratoryInvestigationCodedResult() {
+        final var labResult = new LabResult(List.of(
+            "ignore me",
+            "RSL+CV+::375211000000108:921::Bowel cancer screening programme FOB test normal (finding)",
+            "ignore me"
+        ));
+        var labResultInvestigationResult = assertThat(labResult.getInvestigationResult()).isPresent();
```

Comment:
Personally I think it reads better if you name these intermediate assertion objects as `assertX`. What do you think?
```suggestion
        var assertInvestigationResult = assertThat(labResult.getInvestigationResult()).isPresent();
```
e.g.
```
assertInvestigationResult.isEqualTo(expectation);
```

Reply:
Yeah I agree. I would later read this thinking it's a normal `LaboratoryInvestigationResult` type (and not `OptionalAssert<LaboratoryInvestigationResult>` type)